WPMedia/fusion-news-theme-blocks
# Pen 1541 dfp ad

## Description
- update logic in ad slot while `websiteAdPath` is not set under `blocks.json` in theme repo
## Jira Ticket
- [PEN-1541](https://arcpublishing.atlassian.net/browse/PEN-1541)

## Acceptance Criteria
There would not be an extra / so the sectionID would not be combined with the dfpID

## Test Steps
1. In `Fusion-News-Theme`, pull the latest master branch and in `blocks.json`, update default site's `websiteAdPath` to `"`
![Screen Shot 2020-11-24 at 2 18 09 PM](https://user-images.githubusercontent.com/4411687/100157764-f741fd80-2e5f-11eb-893d-048c7e88f906.png)

2. Go to Fusion-news-theme-blocks directory, run `rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..` on this repo

3. Run `npx fusion start -f -l @wpmedia/ads-block` under `Fusion-News-Theme` directory

4. Add `Google Ad- Arc Block` to your local `homepage`, set the customfield as `leaderoard_large`

5. Update page, then `Stage & Publish` Page

6. Open [http://localhost/pf/homepage/?_website=the-sun](http://localhost/pf/homepage/?_website=the-sun) in chrome

7. Open Chrome's Developer tool, click on "Network" tab and filter by "dfp"

8. look for the "iu_parts" and there should not have any slot information
![Screen Shot 2020-11-24 at 2 33 30 PM](https://user-images.githubusercontent.com/4411687/100159324-67ea1980-2e62-11eb-8857-189523266d70.png)

9. The ad should render as is. 

## Effect Of Changes
### Before



### After


## Dependencies or Side Effects
- Restart `Fusion-News-Theme` toggle between the blocks.json's websiteAdPath) to see the changes 


## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
